assets/css/less/admin.less

```diff
@@ -57,6 +57,10 @@
     }
 }
 
+#adminmenu .wpbdp-orange-text {
+	color: #F47449;
```

Comment:
Is there a specific reason why you didn't add this color to `colors.less`?

Reply:
That file contains only variables. And the color of that link is temporary just for this Black Friday so I didn't really see a reason for it to be a variable.